app/models/user.rb

```diff
@@ -0,0 +1,29 @@
+require 'dm-validations'
+class User
+  include DataMapper::Resource
+
+  has n, :posts, through: Resource
+
+  property :id,       Serial
+  property :username, String
+  property :name,     String
+  property :email,    Text, :required => true, :format => :email_address
+  property :password, BCryptHash, length: 255
+  attr_accessor :password_confirmation
+
+  validates_confirmation_of :password,
+    :message => "Password and confirmation password do not match"
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)